<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="PB_Source">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quelle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_PausePlay">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause/Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Previous">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Next">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_randRep">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand./Rep.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Spare">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Spare</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Back">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_enc0_down">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_enc0_up">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_enc0_sw">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_enc1_up">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_enc1_down">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_enc1_sw">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gV_dis0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>128</width>
     <height>64</height>
    </rect>
   </property>
      <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gV_dis1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>128</width>
     <height>64</height>
    </rect>
   </property>
      <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
